File: ios/BridgeExample/EditorView/EditorViewManager.m
//
//  EditorViewManager.m
//  BridgeExample
//
//  Created by Mauricio Cousillas on 9/11/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(EditorViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(onImageChange, RCTDirectEventBlock)
@end

File: ios/BridgeExample/EditorView/EditorViewManager.swift
//
//  EditorViewManager.swift
//  BridgeExample
//
//  Created by Mauricio Cousillas on 9/11/19.
//  Copyright © 2019 Facebook. All rights reserved.
//

import Foundation

@objc(EditorViewManager)
class EditorViewManager: RCTViewManager {
  override func view() -> UIView! {
    return EditorView()
  }

  override  static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
